commit 5b8f9e48a3f19c8656ede3f9f2f59ea6a96efc41
Author: Markus Gerstel <markus.gerstel@diamond.ac.uk>
Date:   2022-04-12 23:52:48 +0100

    Allow running Relion 3.1 without pipeliner

diff --git a/src/relion/cli/run_pipeline.py b/src/relion/cli/run_pipeline.py
index 9e54daa..b2d7aeb 100644
--- a/src/relion/cli/run_pipeline.py
+++ b/src/relion/cli/run_pipeline.py
@@ -6,7 +6,11 @@ import pathlib
 
 from relion.cryolo_relion_it import cryolo_relion_it, dls_options
 from relion.cryolo_relion_it.cryolo_relion_it import RelionItOptions
-from relion.pipeline import PipelineRunner
+
+try:
+    from relion.pipeline import PipelineRunner
+except ModuleNotFoundError:
+    PipelineRunner = None
 
 
 def run():
@@ -91,6 +95,8 @@ def run():
     if args.version == 3.1:
         cryolo_relion_it.run_pipeline(opts)
     elif args.version == 4:
+        if not PipelineRunner:
+            exit("Relion 4 support requires the CCP-EM pipeliner package")
         suffix = ""
         for movie in (abs_working / "Movies").glob("**/*"):
             if movie.is_file():
